### PR TITLE
[ALP-41] finished checkbox component

### PR DESCRIPTION
**Pull request by michaelfromyeg**
- Able to specify title for checkbox
- Designed according to Figma specifications

### CODE REVIEW

**Comment by michaelfromyeg** [comment]
Simplified the styling (no more nested components) and added functionality for primary and secondary colors

File: aunt-leahs-app/src/components/checkbox.jsx [hunk 1]
```diff
@@ -0,0 +1,85 @@
+import React from "react";
+import {
+  Typography,
```
Comment (thaingocnguyen) [inline aunt-leahs-app/src/components/checkbox.jsx]: This is an unused import

File: aunt-leahs-app/src/components/checkbox.jsx [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+            }}
+          />
+        }
+        label={title}
+      />
+    </div>
+  );
+};
+
+export default Tickbox;
```
Comment (thaingocnguyen) [inline aunt-leahs-app/src/components/checkbox.jsx]: I know this is not the case with all components so far, but it's less confusing for the component to have the same name as the file. The file's name is checkbox while the component's name is Tickbox.

File: aunt-leahs-app/src/components/checkbox.jsx [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+// Two different colors: primary (teal) and secondary (red)
+// Comes with title (label) built in; can be sent in as React Component (e.g., Typography)
+const Tickbox = ({ title, color }) => {
+  const classes = useStyles();
+
+  return (
+    <div className={"checkbox"}>
+      <FormControlLabel
+        control={
+          <Checkbox
```
Comment (thaingocnguyen) [inline aunt-leahs-app/src/components/checkbox.jsx]: Should there be an onChange prop here?

File: aunt-leahs-app/src/components/checkbox.jsx (hunk 2 above)
Comment (michaelfromyeg) [reply aunt-leahs-app/src/components/checkbox.jsx]: Good point! I just didn't want to get it confused with the MaterialUI checkbox; I'll change both to be Tickbox.

File: aunt-leahs-app/src/components/checkbox.jsx (hunk 3 above)
Comment (michaelfromyeg) [reply aunt-leahs-app/src/components/checkbox.jsx]: Yes! Accidentally removed that

File: aunt-leahs-app/src/components/checkbox.jsx (hunk 1 above)
Comment (michaelfromyeg) [reply aunt-leahs-app/src/components/checkbox.jsx]: Removed

File: aunt-leahs-app/src/components/tickbox.jsx [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+// Two different colors: primary (teal) and secondary (red)
+// Comes with title (label) built in; can be sent in as React Component (e.g., Typography)
+const Tickbox = ({ title, color, onChange }) => {
+  const classes = useStyles();
+
+  return (
+    <div className={"checkbox"}>
+      <FormControlLabel
+        control={
+          <Checkbox
```
Comment (codygagnon) [inline aunt-leahs-app/src/components/tickbox.jsx:68]: I think that the Checkbox component needs a checked prop, which takes a boolean state value. Have a look at the Material UI docs: https://material-ui.com/api/checkbox/

**Review by codygagnon — CHANGES_REQUESTED** [review]
Michael, this looks good, but the package.json and package-lock.json files shouldn't be submitted with pull requests. Please see my other comment and make any necessary changes.

**Review by thaingocnguyen — APPROVED** [review]
Nice, should be good after implementing Cody's suggested changes! I'm not sure what the expected interaction should look like with the checkbox (whether the colour around the box should change when you hover over it, etc.) so no comments on the styling except that it looks as expected.

File: aunt-leahs-app/src/components/tickbox.jsx (hunk 4 above)
Comment (michaelfromyeg) [reply aunt-leahs-app/src/components/tickbox.jsx:68]: Oops, totally right! I've added it in in the most recent commit

**Review by codygagnon — APPROVED** [review]

**Pull request merged**